### DISCUSSION TITLE
Golang Backend for Remix

### DISCUSSION THREAD

**Discussion by icommit** [remix-run/remix#5913] [Q&A] [answered]
Hello Mixers. I'm trying to setup a workflow using Golang as the Server for my Remix app. I've seen there's one available for Nodejs but I'm not sure how to go about doing the same for Golang or whether this is possible at all with Remix. Thanks.

**Comment by sergiodxa** [comment, marked as answer]
Remix needs a server side JS runtime, if you could run JS somehow in Go it must be possible to add a server adapter.

Usually to use another language for the backend you run both Remix and the other backend, and fetch the backend from your Remix loaders and actions.
